beavis-ci/beavis-ci.sh

```diff
@@ -19,6 +19,7 @@
 #   -u --username GITHUB_USERNAME, defaults to the environment variable
 #   -k --key      GITHUB_API_KEY, defaults to the environment variable
 #   -b --branch   Test the notebooks in a dev branch. Outputs still go to "rendered"
+#   -r --repo     Specify the repo name, default to LSSTDESC/DC2-analysis
```

Comment:
Nice! When we split beavis out, we'll need the default repo to be read from the $cwd, with, eg `repo=$( git config --get remote.origin.url | cut -d':' -f2 )`